### PR TITLE
VUE-362 VUE-396 Remove lodash omit and pick

### PR DESCRIPTION
**Pull request by eddieferrer**
* Removed lodash methods omit and pick in favor of native JS

VUE-362
VUE-396

### CODE REVIEW

File: src/api/fixtures/LoanSearchCriteria.js [hunk 1]
```diff
@@ -29,22 +28,15 @@ export function getCacheableCriteria(criteria) {
...
 		filters: getInputFilters(filters),
 	};
 }
 
 // Return criteria that can be used in a loan search
-export function getSearchableCriteria({ filters, ...criteria }) {
-	const cleanCriteria = _pick(criteria, [
-		'queryString'
-	]);
+export function getSearchableCriteria({ filters }) {
```
Comment (eddieferrer) [inline src/api/fixtures/LoanSearchCriteria.js:38]: This function is referenced in a single place, and there it is destructured to only use filters

File: src/pages/Checkout/CheckoutPage.vue [hunk 2]
```diff
@@ -547,11 +545,13 @@ export default {
 			const transactionData = {
 				transactionId: numeral(transactionId).value(),
 				itemTotal: this.totals.itemTotal,
-				loans: _map(this.loans, loan => {
-					return _pick(loan, ['__typename', 'id', 'price']);
+				loans: this.loans.map(loan => {
+					const { __typename, id, price } = loan;
+					return { __typename, id, price };
```
Comment (mcstover) [inline src/pages/Checkout/CheckoutPage.vue:550]: did you see any difference in the return value if there are no loans present?

File: src/pages/Checkout/CheckoutPage.vue [hunk 3]
```diff
@@ -547,11 +545,13 @@ export default {
...
 				}),
-				donations: _map(this.donations, donation => {
-					return _pick(donation, ['__typename', 'id', 'price']);
+				donations: this.donations.map(donation => {
+					const { __typename, id, price } = donation;
+					return { __typename, id, price };
```
Comment (mcstover) [inline src/pages/Checkout/CheckoutPage.vue:554]: Same here...So long as they are gracefully handled when missing these should be fine.

File: src/pages/Checkout/CheckoutPage.vue (hunk 2 above)
Comment (eddieferrer) [reply src/pages/Checkout/CheckoutPage.vue:550]: These will both return an empty array

**Pull request merged**